docs/orleans/host/configuration-guide/serialization-configuration.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Serialization configuration in Orleans
+description: Learn how to configure serialization in .NET Orleans.
+ms.date: 12/15/2022
+uid: orleans-serialization-configuration
+zone_pivot_groups: orleans-version
+---
+
+# Serialization configuration in Orleans
+
+<!-- markdownlint-disable MD044 -->
+:::zone target="docs" pivot="orleans-7-0"
+<!-- markdownlint-enable MD044 -->
+
+The configuration of serialization in Orleans is a crucial part of the overall system design. While Orleans provides reasonalble defaults, you can configure serialization to suit your apps' needs. For sending data between hosts, <xref:Orleans.Serialization?displayProperty=fullName> supports delegating to other serializers, such as [Newtonsoft.Json](https://www.nuget.org/packages/Newtonsoft.Json) and [System.Text.Json](https://www.nuget.org/packages/System.Text.Json). You can add support for other serializers by following the pattern set by those implementations. For grain storage it is preferential to use <xref:Orleans.Storage.IGrainStorageSerializer> to configure a custom serializer.
+
+## Configure Orleans to use `System.Text.Json`
+
+Configuring Orleans to use `System.Text.Json` to serialize a subset of types is similar to configuring Orleans to serialize types using `Newtonsoft.Json`, except that the package and configuration methods are different:
```

Comment:
This forward-references the Newtonsoft.Json section, but they won't have read that yet